52/abidmunirmalik/pomodoro.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime, timedelta
+import time 
+import subprocess as sp
+import sys
+
+POMODORO_TIMER_MINUTES = sys.argv[1]
+if int(sys.argv[1]) < 1 or int(sys.argv[1]) > 20:
+    print("timer range between 1 - 20")
+    exit(1)
+elif len(POMODORO_TIMER_MINUTES) == 2:
+    POMODORO_TIMER_MINUTES = POMODORO_TIMER_MINUTES + ':00'
+elif len(POMODORO_TIMER_MINUTES) == 1:
+    POMODORO_TIMER_MINUTES = '0'+POMODORO_TIMER_MINUTES + ':00'
```

Comment:
feel this can be simplified, doing ':00' twice, same for "timer range ..." error, should been used once